.npmignore

```diff
@@ -1,3 +1,3 @@
-.travis.yml
-History.md
 test
+.github
+CHANGELOG.md
```

Comment:
I believe `CHANGELOG.md` is automatically taken by npm (ignoring it is ineffective)